server/lib/email.js

```diff
@@ -226,6 +227,11 @@ const getNotificationLabel = (template, recipients) => {
   return notificationTypeLabels[template];
 };
 
+const isWhitelistedDomain = email => {
+  const domain = email.toString().split('@');
```

Comment:
I'm not sure we need that, do we?

```suggestion
  const domain = email.split('@');
```

Reply:
Made changes :) 

It was not needed 😊